src/lib/commandBuilder.ts

```diff
@@ -118,10 +118,9 @@ export const exit = (
   statusCode: number
 ): Promise<void> => {
   return new Promise(resolve => {
-    log.end();
-    setTimeout(() => {
+    log.info("--end log--", () => {
```

Comment:
@dennisseah so `--end log--` will be at the end of each call in the log regardless of success?

Reply:
> @dennisseah so `--end log--` will be at the end of each call in the log regardless of success?

correct